test_rules_scala.sh

```diff
@@ -19,10 +19,10 @@ $runner bazel build src/... test/...
 $runner bazel test "${test_output_flag}" src/... test/...
 $runner bazel test "${test_output_flag}" third_party/...
 $runner bazel build --extra_toolchains=//test/toolchains:high_level_transitive_deps_strict_deps_error -- test/...
-$runner bazel build --extra_toolchains=//scala:minimal_direct_source_deps -- test/...
+$runner bazel build --extra_toolchains=//scala:minimal_direct_source_deps --build_tag_filters=-no-strict-deps -- test/...
 #$runner bazel build --extra_toolchains=//test/toolchains:high_level_transitive_deps_strict_deps_error --all_incompatible_changes -- test/...
 $runner bazel test "${test_output_flag}" --extra_toolchains=//test/toolchains:high_level_transitive_deps_strict_deps_error -- test/...
-$runner bazel test "${test_output_flag}" --extra_toolchains=//scala:minimal_direct_source_deps -- test/...
+$runner bazel test "${test_output_flag}" --extra_toolchains=//scala:minimal_direct_source_deps --build_tag_filters=-no-strict-deps --test_tag_filters=-no-strict-deps -- test/...
```

Comment:
I don't love that I had to do this, but afaik there isn't a way to disable "strict deps" on a target level like you can with "unused deps"